opensearch_py_ml/ml_models/ml_commons_utils.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# URL of the license file for the OpenSearch project
+LICENSE_URL = "https://github.com/opensearch-project/opensearch-py-ml/raw/main/LICENSE"
+
+# Name of the function used for sparse encoding
+SPARSE_ENCODING_FUNCTION_NAME = "SPARSE_ENCODING"
```

Comment:
Let's use this file: https://github.com/opensearch-project/opensearch-py-ml/blob/main/opensearch_py_ml/ml_commons/ml_common_utils.py ?